recipes/hyperscan/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
```

Comment:
```suggestion
set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR} ${CMAKE_MODULE_PATH})
```

Reply:
Do not remove cmake generator, it's fine to have both cmake and cmake_find_package.

Reply:
Yeah, I actually forgot that `conanbuildinfo.cmake` contains the build settings, compiler flags and such, therefore I thought it was redundant. Updated the suggestions accordingly. Thanks.

Reply:
The docs for the `cmake_find_package` generator say: 

> Moreover, this also adjusts CMAKE_MODULE_PATH and CMAKE_PREFIX_PATH to the values declared by the package in cpp_info.buildirs, so modules in those directories can be found.

So is the setting of CMAKE_MODULE_PATH in the recipes CMakeLists.txt (line 6 in your suggestion) really neccesary?

Reply:
No it's not